packages/fxa-auth-server/fxa-oauth-server/config/dev.json

```diff
@@ -141,7 +141,8 @@
       "imageUri": "",
       "redirectUri": "urn:ietf:wg:oauth:2.0:oob",
       "trusted": true,
-      "canGrant": true
+      "canGrant": true,
+      "publicClient": true
```

Comment:
make sure to change this in fxa-dev?

Reply:
Good catch, will do.

Reply:
https://github.com/mozilla/fxa-dev/pull/469